packages/core/src/components/table/table.css

```diff
@@ -0,0 +1,122 @@
+.hds-table-container {
+  overflow-x: auto;
+}
+
+.hds-table-container:focus {
+  outline: var(--color-coat-of-arms) 3px solid;
+}
+
+.hds-table {
+  border-collapse: collapse;
+  width: 100%;
+  line-height: 1.5;
+}
+
+.hds-table--dark {
+  --background-color: var(--color-bus);
+}
+
+.hds-table--light {
+  --background-color: var(--color-silver);
+}
+
+.hds-table th {
+  border-bottom: 1px solid var(--color-black-20);
+  white-space: nowrap;
+  text-align: left;
+  font-weight: 500;
+  font-size: var(--fontsize-heading-xxs);
+  padding: var(--spacing-xs) var(--spacing-m);
+}
```

Comment:
Suggestion: would it be easier to maintain if default styles and variant styles would be in groups like in the button.css?